### ISSUE TITLE
RCS1074 not being detected

### ISSUE THREAD

**Issue by mahgo** [JosefPihrt/Roslynator#893]
**Product and Version Used**: 
Roslynator.Analyzers 4.1.0

**Steps to Reproduce**:
Set RCS1074 to error or similar in .editorconfig
```
dotnet_diagnostic.RCS1074.severity=error
```

Create following class with constructor:
```csharp
public class Foo
{
    public Foo()
    {
    }
}
```

**Actual Behavior**:
RCS1074 not detected

**Expected Behavior**:
RCS1074 detected

**Comment by JosefPihrt** [comment]
Hi,

I'm not able to reproduce this bug.

**Comment by mahgo** [comment]
@JosefPihrt I've tried replicating it in a new project/solution and it looks like it isn't an issue there. So I'll have to look into more why it's just an issue in my main project/solution.
